data/data/aws/vpc/common.tf

```diff
@@ -7,6 +7,8 @@ data "aws_availability_zones" "azs" {}
 
 // Only reference data sources which are gauranteed to exist at any time (above) in this locals{} block
 locals {
+  lb_base_name = "${replace(var.cluster_name, "/[^A-Za-z0-9-]/", "-")}"
```

Comment:
nit: Is that last hyphen in the regex necessary?

Reply:
No.  As it stands, it is "replace all but {legal chars, including hyphens} with hyphens".  Without the trailing `-`, it wold be replacing the input `-` with `-`.  You'd want the trailing `-` if the replacement was an empty string, etc.  Did tou want me to change it?

Reply:
Oh, duh. Ignore me.